### DISCUSSION TITLE
csv-importer: Adding Pre-made configuration file

### DISCUSSION THREAD

**Discussion by nebula2** [firefly-iii/firefly-iii#4764] [Firefly III]
Hello :)

I was just wondering if there is a possibility to permanently add my json configuration files to this dropdown menu here:

![image](https://user-images.githubusercontent.com/5841525/117691799-982d3680-b1bc-11eb-8237-136d7cf1c08e.png)

If I know the location where this is looking at I can mount the drive in my yml. A feature request would be to upload a config permanently through the web ui (but that is not something I want to trigger with this question)

~~btw: the link to check out the documentation is not working [https://firefly-iii.gitbook.io/firefly-iii-csv-importer/importing-data/upload](https://firefly-iii.gitbook.io/firefly-iii-csv-importer/importing-data/upload)~~

Thank you for your time!

**Comment by JC5** [comment]
What version of the CSV importer are you running?

**Reply by nebula2** [reply]
2.2.3 (sadly a bit outdated at the moment but I could upgrade anytime).

**Reply by nebula2** [reply]
Just updated to 2.5.1 and the link works just fine

**Comment by JC5** [comment]
https://docs.firefly-iii.org/csv/help/config/

This may help? Let me know!

**Reply by nebula2** [reply]
yes this helps. Why haven't I found this myself >.< 
Thank you!
